src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
```

Comment:
>> This would be going even further; we'd get automatically defined all ES6 globals & syntax which not all our supported browsers implement.
>
> Just to clarify, are you referring to jQuery 3 or all versions?

To all jQuery versions currently in development, i.e. `jQuery 3.0.0` & `jQuery Compat 3.0.0`.